bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -104,6 +105,7 @@ public void getTibberParameters() {
         try {
             httpHeader.put("cache-control", "no-cache");
             httpHeader.put("content-type", JSON_CONTENT_TYPE);
+            httpHeader.put("user-agent", "openHAB/3.x com.tibber/1.8.3");
```

Comment:
Is there no way of getting the openHAB/binding version dynamically at runtime? This would need touching at major version upgrades aswell as on every change of this binding, that is not ideal.

Reply:
Suggestion how to would be appreciated.

Reply:
You can learn from some of the other bindings, a quick search for "user-agent" leads to this:
https://github.com/openhab/openhab-addons/blob/cdff5a06e7e9b719b9d3bc484c52012eeed30ba2/bundles/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/handler/WebserviceHandler.java#L89

I assume you only need to find a way to get the openHAB version there aswell.... Maybe you'll find some example aswell.

Reply:
Added dynamic bundle version for "user-agent".